update_ocds_publishers.py

```diff
@@ -0,0 +1,76 @@
+import json
+from pathlib import Path
+
+import click
+import pandas as pd
+
+
+@click.command()
+@click.argument("filename")
+def update_ocds_publishers(filename):
+    """
+    Update the list of OCDS publishers for each country listed in data/oc-status/_index.json using a CSV file.
+
+    The CSV file must be exported from the OCP's MEL1 sheet.
+    """
+    # The columns name and values must match the MEL 1 spreadsheet columns name and values
+    # https://docs.google.com/spreadsheets/d/1NuGNEHpNFxrK-Vf0zGTWVEjrgqt2nYmKDbGN9FLp-k4/edit?gid=1539522696#gid=1539522696
+    non_mvp_reason_column = "Reason for non-MVP status"
+    date_first_mvp_column = "Date of first MVP"
+
+    publishers = pd.read_csv(
+        filename,
+        usecols=[
+            "Publisher or system name (see note if different from organization)",
+            'Country (or "Multiple")',
+            "Conforms",
+            "Status",
+            non_mvp_reason_column,
+            date_first_mvp_column,
+            "URL",
+            "Standard",
+        ],
+        keep_default_na=False,
+    )
+    # Include only retrievable OCDS publishers
+    publishers = publishers[
+        (publishers["Standard"] == "OCDS")
+        & (publishers["Conforms"] == "Yes")
+        & (publishers["Status"] != "Implementing")
+        & (~publishers[non_mvp_reason_column].str.startswith("Retrievable"))
+    ]
+
+    with Path("data/oc-status/_index.json").open() as countries_file:
+        countries = json.load(countries_file)
```

Comment:
We typically just use `f`, unless we're working with multiple file handles.

```suggestion
    with Path("data/oc-status/_index.json").open() as f:
        countries = json.load(f)
```